src/utils/perfherder/chartJs/perfherderFormatter.js

```diff
@@ -12,6 +12,33 @@ const generateInitialOptions = (meta) => {
   return {
     reverse: higherIsBetter,
     scaleLabel: higherIsBetter ? 'Score' : 'Load time',
+    tooltips: {
+      callbacks: {
+        footer: (tooltipItems, data) => {
+          let delta = 0;
+          tooltipItems.map((tooltipItem) => {
```

Comment:
You don't need to map over `tooltipItems` when there will only be one item in the array (since you are selecting one data point). In this situation, just access it via index `tooltipItems[0]` and remove the arrow function.

Reply:
Updated!